actions/pack_action.go

```diff
@@ -30,10 +31,38 @@ type PackAction struct {
 	File             string
 }
 
+
+func (pf *PackAction) Verify(context *debos.DebosContext) error {
+
+	if len(pf.File) == 0 {
+		return fmt.Errorf("Please add a 'file' property.")
+	}
+
+	archive, err := debos.NewArchive(pf.File)
```

Comment:
Would make sense to add a NewTarArchive(compression) function rather then using NewArchive (which mostly looks at which type of archive it should be).